Detectors/FIT/FDD/dcsmonitoring/workflow/fdd-dcs-sim-workflow.cxx

```diff
@@ -23,6 +23,7 @@ o2::framework::WorkflowSpec defineDataProcessing(o2::framework::ConfigContext co
   // for testing, we use less DPs than the official ones
   dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"FDD/SIDE_A/HV_A9/[I,V]MON", 250, 350});
   dphints.emplace_back(o2::dcs::test::DataPointHint<uint>{"FDD/PM/SIDE_A/PMT_A_9/ADC[0,1]_BASELINE", 30, 150});
+  dphints.emplace_back(o2::dcs::test::DataPointHint<double>{"FDD/Trigger1_Central/CNT_RATE", 0, 1000000});
```

Comment:
Assuming 1000000 is the max expected value, I'd say it should be increased, at least by a factor of 5. Same for FT0 and FV0

Reply:
Ok I changed it. These are only the range of randomly generated values if one wants to simulate ADAPOS input locally. The production workflow sets no limits on anything.